frontend/src/styles/challenge-creation-code-editor.sass

```diff
@@ -0,0 +1,21 @@
+@import "screen-sizes-variables"
```

Comment:
all challenge related code should be a part of the challenge module

Reply:
moved

Reply:
moved style file in the **challenge module/styles**